lib/util.js

```diff
@@ -501,22 +508,47 @@ module.exports = {
           // TODO: have to add auth here (but first, auth to be put into the openapi tree)
         });
 
+        // If a folder has only one child which is a folder then we collapsed the child folder
```

Comment:
Here. If a folder has only one child, should check for the immediate requests too